### PR TITLE
enhance: format list privilege group result

### PR DESCRIPTION
**Pull request by shaoting-huang**
issue: https://github.com/milvus-io/milvus/issues/37031

### CODE REVIEW

File: pymilvus/milvus_client/milvus_client.py [hunk 1]
```diff
@@ -1219,16 +1219,19 @@ def list_privilege_groups(
...
             MilvusException: If anything goes wrong.
         """
         conn = self._get_connection()
-        pgs = conn.list_privilege_groups(timeout=timeout, **kwargs)
-        ret = {}
-        for pg in pgs:
-            ret[pg.group_name] = [p.name for p in pg.privileges]
+        try:
+            res = conn.list_privilege_groups(timeout=timeout, **kwargs)
+        except Exception as ex:
```
Comment (XuanYang-cn) [inline pymilvus/milvus_client/milvus_client.py:1230]: maybe log the exception

**Review by XuanYang-cn — CHANGES_REQUESTED** [review]

File: pymilvus/milvus_client/milvus_client.py (hunk 1 above)
Comment (shaoting-huang) [reply pymilvus/milvus_client/milvus_client.py:1230]: added

**Comment by XuanYang-cn** [comment]
> issue: [milvus-io/milvus#37031](https://github.com/milvus-io/milvus/issues/37031)
> 
> ```
> >>> role.list_privilege_groups()
> PrivilegeGroupInfo groups:
> - PrivilegeGroupItem: <privilege_group:privilege_group>, <privileges:('Insert', 'Release')>
> - PrivilegeGroupItem: <privilege_group:ClusterReadWrite>, <privileges:('ListDatabases', 'SelectOwnership', 'SelectUser', 'DescribeResourceGroup', 'ListResourceGroups', 'FlushAll', 'TransferNode', 'TransferReplica', 'UpdateResourceGroups')>
> - PrivilegeGroupItem: <privilege_group:ClusterAdmin>, <privileges:('ListDatabases', 'SelectOwnership', 'SelectUser', 'DescribeResourceGroup', 'ListResourceGroups', 'FlushAll', 'TransferNode', 'TransferReplica', 'UpdateResourceGroups', 'BackupRBAC', 'RestoreRBAC', 'CreateDatabase', 'DropDatabase', 'CreateOwnership', 'DropOwnership', 'ManageOwnership', 'CreateResourceGroup', 'DropResourceGroup', 'UpdateUser')>
> - PrivilegeGroupItem: <privilege_group:CollectionReadOnly>, <privileges:('Query', 'Search', 'IndexDetail', 'GetFlushState', 'GetLoadState', 'GetLoadingProgress', 'HasPartition', 'ShowPartitions', 'DescribeCollection', 'DescribeAlias', 'GetStatistics', 'ListAliases')>
> - PrivilegeGroupItem: <privilege_group:CollectionAdmin>, <privileges:('Query', 'Search', 'IndexDetail', 'GetFlushState', 'GetLoadState', 'GetLoadingProgress', 'HasPartition', 'ShowPartitions', 'DescribeCollection', 'DescribeAlias', 'GetStatistics', 'ListAliases', 'Load', 'Release', 'Insert', 'Delete', 'Upsert', 'Import', 'Flush', 'Compaction', 'LoadBalance', 'RenameCollection', 'CreateIndex', 'DropIndex', 'CreatePartition', 'DropPartition', 'CreateAlias', 'DropAlias')>
> - PrivilegeGroupItem: <privilege_group:ClusterReadOnly>, <privileges:('ListDatabases', 'SelectOwnership', 'SelectUser', 'DescribeResourceGroup', 'ListResourceGroups')>
> - PrivilegeGroupItem: <privilege_group:DatabaseAdmin>, <privileges:('ShowCollections', 'DescribeDatabase', 'AlterDatabase', 'CreateCollection', 'DropCollection')>
> - PrivilegeGroupItem: <privilege_group:CollectionReadWrite>, <privileges:('Query', 'Search', 'IndexDetail', 'GetFlushState', 'GetLoadState', 'GetLoadingProgress', 'HasPartition', 'ShowPartitions', 'DescribeCollection', 'DescribeAlias', 'GetStatistics', 'ListAliases', 'Load', 'Release', 'Insert', 'Delete', 'Upsert', 'Import', 'Flush', 'Compaction', 'LoadBalance', 'RenameCollection', 'CreateIndex', 'DropIndex', 'CreatePartition', 'DropPartition')>
> - PrivilegeGroupItem: <privilege_group:DatabaseReadOnly>, <privileges:('ShowCollections', 'DescribeDatabase')>
> - PrivilegeGroupItem: <privilege_group:DatabaseReadWrite>, <privileges:('ShowCollections', 'DescribeDatabase', 'AlterDatabase')>
> ```

@shaoting-huang Please move the codes outside of the description. The description are used as git commit messages, this code is too long for a git commit messages

**Comment by shaoting-huang** [comment]
the format should be
```

```
>>> role.list_privilege_groups()
PrivilegeGroupInfo groups:
- PrivilegeGroupItem: <privilege_group:privilege_group>, <privileges:('Insert', 'Release')>
- PrivilegeGroupItem: <privilege_group:ClusterReadWrite>, <privileges:('ListDatabases', 'SelectOwnership', 'SelectUser', 'DescribeResourceGroup', 'ListResourceGroups', 'FlushAll', 'TransferNode', 'TransferReplica', 'UpdateResourceGroups')>
- PrivilegeGroupItem: <privilege_group:ClusterAdmin>, <privileges:('ListDatabases', 'SelectOwnership', 'SelectUser', 'DescribeResourceGroup', 'ListResourceGroups', 'FlushAll', 'TransferNode', 'TransferReplica', 'UpdateResourceGroups', 'BackupRBAC', 'RestoreRBAC', 'CreateDatabase', 'DropDatabase', 'CreateOwnership', 'DropOwnership', 'ManageOwnership', 'CreateResourceGroup', 'DropResourceGroup', 'UpdateUser')>
- PrivilegeGroupItem: <privilege_group:CollectionReadOnly>, <privileges:('Query', 'Search', 'IndexDetail', 'GetFlushState', 'GetLoadState', 'GetLoadingProgress', 'HasPartition', 'ShowPartitions', 'DescribeCollection', 'DescribeAlias', 'GetStatistics', 'ListAliases')>
- PrivilegeGroupItem: <privilege_group:CollectionAdmin>, <privileges:('Query', 'Search', 'IndexDetail', 'GetFlushState', 'GetLoadState', 'GetLoadingProgress', 'HasPartition', 'ShowPartitions', 'DescribeCollection', 'DescribeAlias', 'GetStatistics', 'ListAliases', 'Load', 'Release', 'Insert', 'Delete', 'Upsert', 'Import', 'Flush', 'Compaction', 'LoadBalance', 'RenameCollection', 'CreateIndex', 'DropIndex', 'CreatePartition', 'DropPartition', 'CreateAlias', 'DropAlias')>
- PrivilegeGroupItem: <privilege_group:ClusterReadOnly>, <privileges:('ListDatabases', 'SelectOwnership', 'SelectUser', 'DescribeResourceGroup', 'ListResourceGroups')>
- PrivilegeGroupItem: <privilege_group:DatabaseAdmin>, <privileges:('ShowCollections', 'DescribeDatabase', 'AlterDatabase', 'CreateCollection', 'DropCollection')>
- PrivilegeGroupItem: <privilege_group:CollectionReadWrite>, <privileges:('Query', 'Search', 'IndexDetail', 'GetFlushState', 'GetLoadState', 'GetLoadingProgress', 'HasPartition', 'ShowPartitions', 'DescribeCollection', 'DescribeAlias', 'GetStatistics', 'ListAliases', 'Load', 'Release', 'Insert', 'Delete', 'Upsert', 'Import', 'Flush', 'Compaction', 'LoadBalance', 'RenameCollection', 'CreateIndex', 'DropIndex', 'CreatePartition', 'DropPartition')>
- PrivilegeGroupItem: <privilege_group:DatabaseReadOnly>, <privileges:('ShowCollections', 'DescribeDatabase')>
- PrivilegeGroupItem: <privilege_group:DatabaseReadWrite>, <privileges:('ShowCollections', 'DescribeDatabase', 'AlterDatabase')>
```

```

**Review by XuanYang-cn — APPROVED** [review]

**Pull request merged**